PhotoPoints/app/src/main/res/layout/fragment_details.xml

```diff
@@ -0,0 +1,106 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Also, is it possible that you forgot to commit the java Fragment class that will control this View?

Reply:
Ah, for some reason it was unversioned and not automatically included in my commits

Reply:
Use "git add *", before committing does the trick :) ! "git commit -am "some-comment"" only commits changed to files that are tracked.